docs/rfcs/merger_matcher/README.md

```diff
@@ -11,47 +11,62 @@ merge them into combined works to aid understanding and searching.
 
 ## Problem Statement
 
-Individual works can be merged to form larger combined works.
+Individual works can be merged to form larger combined works. Combined works can be later split.
+Each individual work has a unique source identifier. The source identifier for the merged works will be one of the source
+identifier of the original works that compose it.
+
 The merging of works is broken into two phases
 
- * a matching/redirect phase that identifies works to be merged and tracks previously identified combined works.
+ * a matching phase that identifies groups of work to be merged.
  * a merging phase that merges works to form new combined works.
    
```

Comment:
We use different phrases for similar things (groups, matcher, links) I suggest sticking to one for clarity.  Linked works feels closest at the moment to me.  So perhaps s/groups of work/linked works/